care/facility/models/asset.py

```diff
@@ -46,11 +46,45 @@ class RoomType(enum.Enum):
         Facility, on_delete=models.PROTECT, null=False, blank=False
     )
 
+    users = models.ManyToManyField(
+        User,
+        through="AssetLocationDutyStaff",
+        related_name="duty_staff",
+        through_fields=("asset_location", "user"),
+        blank=True,
+    )
+
     middleware_address = models.CharField(
         null=True, blank=True, default=None, max_length=200
     )
 
 
+class AssetLocationDutyStaff(models.Model):
```

Comment:
Shouldn't this be inherited from BaseModel instead?

Reply:
I don't think we need other BaseModel fields for this model. These fields are sufficient.

Reply:
How would you track the datetime the user was assigned to a particular location? You'd need the created_date from the BaseModel right?

Reply:
@sainak thoughts?